packages/connect/src/device/Device.ts

```diff
@@ -467,7 +483,7 @@ export class Device extends TypedEmitter<DeviceEvents> {
             .then(() => {
                 if (wasUnacquired && !this.isUnacquired()) {
                     this.lifecycle.emit(DEVICE.CONNECT);
-
+                    // TODO: maybe there could be a `capabilities` like `"Capability_PUSH"` so we subscribe only when necessary.
```

Comment:
I had the same idea, semantically it makes sense but on the other hand - Capabilities are sent by firmware and I am not sure if it is a good idea to create artificial capabilities 🤔